charter.html

```diff
@@ -334,8 +348,6 @@ <h2 id="participation">
         <p>
           The group also welcomes non-Members to contribute technical submissions for consideration upon their agreement to the terms of the <a href="https://www.w3.org/Consortium/Patent-Policy/">W3C Patent Policy</a>.
         </p>
-        <p>Participants in the group are required (by the <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">W3C Process</a>) to follow the
```

Comment:
Are you intentionally removing the code of ethics and professional conduct from the charter here @jwrosewell ?

Reply:
It will be dependent on the W3C Process. If the Process CG remove it then it'll be removed from all Working Groups that don't reference it explicitly. If it changes then there will be a member review. At the moment it is references in the W3C Process and doesn't need to be duplicated in each charter.

Reply:
I think that if there is a process change we can change the charter but until that point I don't see a clear reason to exclude it, even if it is implicitly part of the charter by dint of the W3C process it can't hurt to have it in here to remind folks that this is indeed part of our process.